config/samples/kustomization.yaml

```diff
@@ -1,4 +1,6 @@
 # SPDX-License-Identifier: MIT
 # SPDX-FileCopyrightText: (c) Copyright 2023 Advanced Micro Devices, Inc.
+
+# Bundled into CSV's `alm-examples`
```

Comment:
Could you elaborate on this? Or, if possible could it be possible to link it to a filepath like `config/samples/onload/kustomization.yaml` rather than here? I'm not completely comfortable with having the root of all samples only point to onload when when there are other types of samples included (sfptpd, sfnettest, etc.)

Reply:
If its an external constraint then i don't have a real issue with it.